mkdocs.yml

```diff
@@ -130,10 +130,11 @@ nav:
   - 'Signing':                        'Signing.md'
   - 'File attachments':               'FileAttachments.md'
 - 'Mixing other libs':
-  - 'Existing PDFs':                  'ExistingPDFs.md'
```

Comment:
There is also a reference to this file in `docs/index.md`, could you please update it so that the hyperlink does not break?

Reply:
Done!